.travis.yml

```diff
@@ -0,0 +1,55 @@
+---
+sudo: true
+
+addons:
+  apt:
+    packages:
+      - docker-ce
+      - python
+      - realpath
+      - ruby
+
+language: go
+
+env:
+  global:
+    - GO_METALINTER_VERSION="v2.0.11"
+
+go:
+  # Kubernetes minimum_go_version can be found in this file:
+  # https://github.com/kubernetes/kubernetes/blob/master/hack/lib/golang.sh
+  - master
+  - stable
+  - "1.10"
+  - "1.9"   # kubernetes 1.10 min was 1.9.1
+
+install:
+  - gem install asciidoctor mdl
+  - pip install --user --upgrade pip
+  - pip install --user yamllint
+  # install dep
+  - >
+    curl -L
+    https://raw.githubusercontent.com/golang/dep/master/install.sh
+    | sh
+  # install gometalinter
+  - >
+    curl -L
+    'https://raw.githubusercontent.com/alecthomas/gometalinter/master/scripts/install.sh'
+    | bash -s -- -b ${TRAVIS_HOME}/gopath/bin "${GO_METALINTER_VERSION}"
+
+script:
+  # Lint text-like files
+  - scripts/pre-commit.sh --require-all
+  - make vendor-install
+  # Lint golang sources
+  - >
+    gometalinter -j4 --sort=path --sort=line --sort=column
```

Comment:
how about doing `metalinter` check after `build`?

Reply:
I assume the concern is that you'd like to know if the build fails?
metalinter and the build will both run... Travis continues to the end of the stage even if there are failures.

Reply:
my concern was if a build fails no need to do any tests further. 
if the jobs continue, then we don't have any issue in the order in which we run the scripts